src/plugins/intel_gpu/src/graph/impls/onednn/fully_connected_onednn.cpp

```diff
@@ -133,33 +135,40 @@ struct fully_connected_onednn : typed_primitive_onednn_impl<fully_connected, dnn
 
         if (!prim->bias.empty()) {
             auto bias_md = onednn::layout_to_memory_desc(impl_params.get_input_layout(2), dnnl::memory::format_tag::any, true);
-            return std::make_shared<dnnl::inner_product_forward::desc>(
+            return std::make_shared<dnnl::inner_product_forward::primitive_desc>(
+                engine.get_onednn_engine(),
                 dnnl::prop_kind::forward_inference,
                 input_md,
                 weights_md,
                 bias_md,
-                output_md);
+                output_md,
+                attr);
         } else {
-            return std::make_shared<dnnl::inner_product_forward::desc>(
+            return std::make_shared<dnnl::inner_product_forward::primitive_desc>(
+                engine.get_onednn_engine(),
                 dnnl::prop_kind::forward_inference,
                 input_md,
                 weights_md,
-                output_md);
+                output_md,
+                attr);
         }
     }
 
 public:
     void save(BinaryOutputBuffer& ob) const override {
+#if 0
         parent::save(ob);
 
         ob << make_data(&_desc->data, sizeof(dnnl_inner_product_desc_t));
 
         std::vector<uint8_t> prim_cache;
         prim_cache = _prim.get_cache_blob();
         ob << prim_cache;
+#endif
```

Comment:
I think we need to throw exception here to clearly show that it is not supported.